.dockerignore

```diff
@@ -1,5 +1,5 @@
 # User data is mounted into the container.
-data/
+#-data/
 
```

Comment:
We should keep the data in the ignore, otherwise when I build an image locally it will copy the data as well.